react-api-client/src/runs/useAddCameraImageSettingsToRunMutation.ts

```diff
@@ -0,0 +1,60 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { addCameraImageSettingsToRun } from '@opentrons/api-client'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type { UseMutateFunction, UseMutationResult } from 'react-query'
+import type {
+  CameraResponse,
+  CameraSettings,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export interface AddCameraImageSettingsToRunParams {
+  runId: string
+  settings: CameraSettings
+}
+
+export type UseAddCameraSettingsToRunMutationResult = UseMutationResult<
+  CameraResponse,
```

Comment:
```suggestion
  CameraSettingsResponse,
```
ditto, applies to all usages in file